addons/mail/models/mail_render_mixin.py

```diff
@@ -203,15 +206,19 @@ def _prepend_preview(self, html, preview):
         if preview:
             preview = preview.strip()
 
+        template_instructions = parse_inline_template(preview)
+        preview_markup = Markup('')
+        for string, expression in template_instructions:
+            preview_markup += Markup(r'{}' + f'<t t-out="{expression}"/>').format(string)
```

Comment:
This might break if the expression contains `"` no ?

Reply:
Fixed.

Reply:
Shouldn't HTML escape also `<>` ?
What happen if you have `"{{'<b>test</b>'}}"` in your preview ?

Reply:
Additionally, could be great to have some unit tests covering this part specifically as we are playing with a lot of error prone constructs here.